packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -1765,11 +1781,17 @@ export interface ControlBarProps {
     styles?: BaseCustomStyles;
 }
 
+// @public (undocumented)
+export function createAzureCommunicationCallAdapter(args: AzureCommunicationCallAdapterArgs): Promise<CallAdapter>;
+
+// @public (undocumented)
+export function createAzureCommunicationCallAdapter(args: AzureCommunicationOutboundCallAdapterArgs): Promise<CallAdapter>;
+
 // @public
-export const createAzureCommunicationCallAdapter: ({ userId, displayName, credential, locator, options }: AzureCommunicationCallAdapterArgs) => Promise<CallAdapter>;
+export function createAzureCommunicationCallAdapterFromClient(callClient: StatefulCallClient, callAgent: CallAgent, targetCallees: (MicrosoftTeamsAppIdentifier | UnknownIdentifier)[], /* @conditional-compile-remove(video-background-effects) */ options?: AzureCommunicationCallAdapterOptions): Promise<CallAdapter>;
```

Comment:
nit: we should do some clean up to remove conditionally compiled comments for features that have GA'ed

Reply:
Should be separate PR's but I agree